product_profile/product.py

```diff
@@ -79,52 +79,51 @@ def _get_profile_fields(self):
                 if field not in fields_to_exclude]
 
     @api.model
-    def _get_profile_data(self, profile_id, filled_fields=None):
+    def _get_vals_from_profile(self, profile_id):
```

Comment:
renaming a method breaks the API and should only occurs when upgrading the addons ( for ex: 8.0 -> 90)